src/store/modules/ui.store.js

```diff
@@ -175,6 +175,17 @@ export default {
          * @type Boolean
          */
         showDragAndDropOverlay: false,
+
+        /**
+         * Flag set to true when the app is opening a new tab.
+         *
+         * This helps us decide to show or not show a warning popup for lost changes if the user
+         * closes the tab. In some cases, we are opening a new tab ourselves (print result) and
+         * don't want this popup to show up.
+         *
+         * @type Boolean
+         */
+        isOpeningNewTab: false,
```

Comment:
I don't like this approach, and would prefer we play around with the closeTab event interception, but I can see why this would be simpler to do this way :) We can go with this if you're satisfied with it :)

Reply:
for now we will live with it, I'll create a task to sort this out for the next release